package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "portfolio",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kavdi/portfolio.git"
+  },
+  "author": "kavdi",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/kavdi/portfolio/issues"
+  },
+  "homepage": "https://github.com/kavdi/portfolio#readme",
+  "dependencies": {
+    "body-parser": "^1.18.1",
+    "express": "^4.15.4",
+    "nodemon": "^1.12.1"
```

Comment:
You do not need `nodemon` as a dependancy. It is a program that should be installed globally that happens to be a node package.